### ISSUE TITLE
read dvd movies

### ISSUE THREAD

**Issue by legacychimera247** [Rafostar/clapper#507]
it would be nice to have the ability to read dvd movies, directly into clapper, love clapper but i would not like to have another app to do (mostly) the same thing (watching movies/videos) so it would be nice to have dvd support integrated :)

**Comment by Rafostar** [comment]
yeah, improving/finishing that part is definitely on the TODO list.

Quick notes what needs to be still done for myself:
- [ ] Fix DVD menu navigation (currently broken in 0.6)
- [ ] Maybe add an Open CD/DVD menu option
- [ ] Make play from Nautilus (file manager) work

**Comment by legacychimera247** [comment]
1 was it working in 0.5?does it work in 0.7?
2 first thing that came into mind when inserted the dvd was to open nautilus, which prompted me to open clapper, so maybe n.3 will suffice?of course having both would be nice, but i suppose launching it via nautilus would suffice for a first implementtion?
3 awesome, now it does ask to open clapper, but then clapper says it's a directory, not a movie

**Comment by sp1ritCS** [comment]
> 1 was it working in 0.5?

yes.

> does it work in 0.7?

not as of now

note that this is only about dvd menu interaction support. You should still be able to play a dvd without such menu by opening `dvd:///dev/sr0` or `dvd:///path/to/mountpoint/` (for isos)

Btw @Rafostar, maybe add support for opening isos of dvds without having to manually loopback mount them to your dvd todo list. VLC has that feature and I find it quite useful.

**Comment by Rafostar** [comment]
Yes, the DVD menu navigation worked previously, but its not working currently due to how UI and its overlays work in new implementation. This will require reworking our sink (something that was planned anyway). It was skipped for 0.6 because GTK4 was still experimenting with their renderers (I do not want to write a new sink for each GTK minor version) and because dvd playback was mostly "hidden" with access by using posted above URIs.

> maybe add support for opening isos of dvds without having to manually loopback mount them to your dvd todo list.

I will add this to not forget to check. Currently unsure how we would handle that. Also, ideally it should be done somehow that it works on all Linux/Windows/macOS.
